### ISSUE TITLE
Tag or button to create directory inside user space

### ISSUE THREAD

**Issue by bipbip007** [SergiX44/XBackBone#213]
Hi,

I was looking after something similar to BoZoN (seems not supported anymore). I found your project. Installation on my personnal server was faster and easy :-)

Thanks for your app.

I use it to share pictures on forum and share files with friends. To avoid to find the full path of all pictures, your url encryption is nice. 

Do you think it could be possible to add a field to add same tag for all files/pictures uploaded in same time ?
and/or possibility to create directories to put all files related to the same projects in same location ?

Thanks in advance.

Bip

**Comment by SergiX44** [comment]
Yeah the tag system could be expanded, maybe adding automatically the username or custom tags based on the upload mime-type.

**Comment by EricGuic** [comment]
+1 for the directories to organize large collections of files (alongside the tags feature which is great !)
